app/views/declaration/AddPersonView.scala.html

```diff
@@ -121,11 +121,14 @@ <h2 class="govuk-heading-m">@messages("responsiblepeople.position_within_busines
 
         @inputCheckbox(
             Checkboxes(
-                hint = Some(
-                    Hint(
-                        content = Text(messages("lbl.select.all"))
-                    )
-                ),
+                fieldset = Some(Fieldset(
+                    legend = Some(Legend(
+                        content = Text(messages("responsiblepeople.position_within_business.h2")),
```

Comment:
You should be able to remove the h2 above and style this legend content to follow the original styling of the h2 (check other Legend components in the service for examples on which classes to use)

By doing that, we can remove duplicate content for screen-readers

Reply:
amended to remove h2 and update class to reflect h2 heading.